phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexScrutinyTool.java

```diff
@@ -505,6 +509,18 @@ public int run(String[] args) throws Exception {
         }
     }
 
+    private void validateTimestamp(Configuration configuration, long ts) {
+        long maxLookBackAge = ScanInfoUtil.getMaxLookbackInMillis(configuration);
+        if (maxLookBackAge != ScanInfoUtil.DEFAULT_PHOENIX_MAX_LOOKBACK_AGE * 1000L) {
```

Comment:
Is there a chance that people coincidentally pick the same Max lookback age as the default one?

Reply:
The default age is the same as "turn off max lookback". It happens to be 0, which makes the 1000L not necessary, but I wanted to keep the discipline of _always_ converting to millis from seconds to avoid accidentally missing it elsewhere.